### DISCUSSION TITLE
Play Protect rates MMRL as possibly malicious

### DISCUSSION THREAD

**Discussion by elhennig** [MMRLApp/MMRL#211] [General]
Since this week, MMRL is rated by Play Protect as possibly malicious. This is on one hand understandable as it is an app that alloes to install modules for a root instance which is not wanted in general. On the other hand similar apps like Mrepo is not rated as malicious although it does the same thing.
What is the actual reason for such rating?

**Comment by DerGoogler** [comment]
Thanks for the report @elhennig,

could you please provide a screenshot? if possible?

**Reply by krokxx03** [reply]
![Screenshot_2025-05-02-11-45-46-949_com miui home](https://github.com/user-attachments/assets/cd031f1c-cd37-43d2-a0ec-54f51db9bcf9)
How to hide it

**Comment by elhennig** [comment]
Herr you are
![Screenshot_20240726-230730](https://github.com/user-attachments/assets/72c2efa2-1ccc-42ba-8e82-937684fffa89)

**Reply by DerGoogler** [reply]
Where you did get the app from?

**Reply by elhennig** [reply]
I installed it from F-Droid/Neo Store, Repo is IzzyOnDroid, Version 2.20.21 from July 25th

**Reply by DerGoogler** [reply]
This may the reason. Google only knows my private sign key and F-Droid uses a different key.

**Comment by TheRealAshik** [comment]
Mrepo vs. this which is better?

**Comment by elhennig** [comment]
They merged, some few weeks ago
